generators/package/templates/_package.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Let's not make this a template. Let's fill values on an object and writeJSON that object.

This way we can extend a pre-existing package.json file and not be destructive.